src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php

```diff
@@ -864,7 +869,13 @@ private function addNewInstance(Definition $definition, $return, $instantiation,
             $inheritanceProxy = sprintf("%s implements \\%s\n{\n%s%s}\n", $class, InheritanceProxyInterface::class, $inheritanceProxy, $this->addServiceOverriddenMethods($id, $definition));
             $class = 'SymfonyProxy_'.md5($inheritanceProxy);
             $this->inheritanceProxies[$class] = $inheritanceProxy;
-            $constructor = $this->container->getReflectionClass($definition->getClass())->getConstructor();
+            $r = $this->container->getReflectionClass($definition->getClass());
```

Comment:
this one is after the call to addServiceOverriddenMethods so it can also be removed